Tests/js/views/services/assets/ez-dashboardblocksviewservice-tests.js

```diff
@@ -168,9 +168,17 @@ YUI.add('ez-dashboardblocksviewservice-tests', function (Y) {
 
         setUp: function () {
             this.rootLocationModel = new Y.Mock();
+            this.app = new Y.Mock();
+            this.user = new Y.Model();
 
+            Y.Mock.expect(this.app, {
+                method: 'get',
+                args: ['user'],
+                returns: this.rootLocationModel
```

Comment:
`get('user')` returns the root Location Model ?

also, you need to add a test to check that `getViewParameters` returns this user.